src/components/CreateVmWizard/CreateVmWizard.js

```diff
@@ -171,7 +172,12 @@ class CreateVmWizard extends React.Component {
 
             onUpdate={({ valid = true, ...updatePayload }) => {
               this.handleListOnUpdate('network', 'nics', updatePayload)
+
+              const isPreventExitChanged = this.wizardStepsMap.network.preventExit === valid
               this.wizardStepsMap.network.preventExit = !valid
+              if (isEmpty(updatePayload) && isPreventExitChanged) {
+                this.forceUpdate()
+              }
```

Comment:
Looks good , but I think that the `===` condition is a little bit confusing and not intuitive
and maybe it can be something like :
`
const newPreventExit = !valid
const isPreventExitChanged = this.wizardStepsMap.network.preventExit !== newPreventExit
this.wizardStepsMap.network.preventExit = newPreventExit
`
IMHO it will improve the readability...
everything else looks good